hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/s3guard/ITestDynamoDBMetadataStore.java

```diff
@@ -921,43 +913,6 @@ private DDBPathMetadata verifyInAncestor(AncestorState state,
     return md;
   }
 
-  @Test
```

Comment:
Why did you remove this test?

Reply:
(a) slow (b) regularly timing out on teardown. We don't need a separate test to show we can provision a table, as we do that in the class setup/teardown anyway.

Reply:
yes, this is slow, and also if we cannot provision a table we will fail other tests. This can go.